include/igl/embree/EmbreeDevice.cpp

```diff
@@ -0,0 +1,9 @@
+#include "EmbreeDevice.h"
+// Implementation
+
+namespace igl {
+    namespace embree {
+     RTCDevice EmbreeDevice::g_device = nullptr;
```

Comment:
Using such globals in a libigl library is problematic, since the .cpp can be included in multiple compilation units. I would recommend following the Meyer singleton approach that I suggested last time. Meyer's singleton pattern has the advantage of being safe to initialize from multiple threads.

Reply:
ok, i fixed that